src/rust/bitbox02-rust/src/hww/api/ethereum/address.rs

```diff
@@ -58,6 +74,28 @@ mod tests {
         assert_eq!(
             from_pubkey_hash(
                 b"\xf4\xc2\x17\x10\xef\x8b\x5a\x5e\xc4\xbd\x37\x80\xa6\x87\xfe\x08\x34\x46\xe6\x7b",
+                pb::AddressCase::Mixed as i32,
+            ),
+            "0xF4C21710Ef8b5a5Ec4bd3780A687FE083446e67B"
+        );
+        assert_eq!(
+            from_pubkey_hash(
+                b"\xf4\xc2\x17\x10\xef\x8b\x5a\x5e\xc4\xbd\x37\x80\xa6\x87\xfe\x08\x34\x46\xe6\x7b",
+                pb::AddressCase::Upper as i32,
+            ),
+            "0xF4C21710EF8B5A5EC4BD3780A687FE083446E67B"
+        );
+        assert_eq!(
+            from_pubkey_hash(
+                b"\xf4\xc2\x17\x10\xef\x8b\x5a\x5e\xc4\xbd\x37\x80\xa6\x87\xfe\x08\x34\x46\xe6\x7b",
+                pb::AddressCase::Lower as i32,
+            ),
+            "0xf4c21710ef8b5a5ec4bd3780a687fe083446e67b"
+        );
+        assert_eq!(
+            from_pubkey_hash(
+                b"\xf4\xc2\x17\x10\xef\x8b\x5a\x5e\xc4\xbd\x37\x80\xa6\x87\xfe\x08\x34\x46\xe6\x7b",
+                3,
             ),
```

Comment:
remove, same as first test

Reply:
but it was the test case when the case variable is not valid(not 0,1,2)